.copyrightignore

```diff
@@ -20,30 +20,32 @@
 #
 # ===========================================================================
 #
-#  THE FOLLOWING ARE EXAMPLES OF SYNTAXES copyrightCheck SUPPORTS:
-#  1.**/jobs/** # path contains "jobs" folder, but not root directory
-#  2./doc#root root level directory and file named "doc"
-#  3./debugtools/**/*.ignore # all .ignore file in the root directory debugtools
-#  4./*.ignore #all root directory .ignore files
-#  5.**/jobs/pipelines/ #pipeline folder follows closely with jobs not in root directory
-#  6.*.ignore #all ignore files only
-#  7.**/jobs/**/pipelines/ #directory path contains both jobs and pipelines not root directory
-#  8.*job*  #all things at contains characters in between stars
+# THE FOLLOWING ARE EXAMPLES OF SYNTAXES copyrightCheck SUPPORTS:
+# 1. **/jobs/** # path contains "jobs" folder, but not root directory
+# 2. /doc # root level directory and file named "doc"
+# 3. /debugtools/**/*.ignore # all .ignore files in the root directory debugtools
+# 4. /*.ignore # all root directory .ignore files
+# 5. **/jobs/pipelines/ # pipeline folder follows closely with jobs not in root directory
+# 6. *.ignore # all ignore files only
+# 7. **/jobs/**/pipelines/ # directory path contains both jobs and pipelines not root directory
+# 8. *job* # all things at contains characters in between stars
 # ===========================================================================
 
-src/bsd/doc
-src/jdk.crypto.ec/share/native/libsunec/impl
-src/linux/doc
-src/solaris/doc
-/test
+/src/bsd/doc/
+/src/jdk.crypto.ec/share/native/libsunec/impl/
+/src/linux/doc/
+/src/solaris/doc/
+/test/
 
+.copyrightignore
```

Comment:
Why does .copyrightignore need to be ignored?

Reply:
Initially I wanted to remove it (in versions where it was present), but outside the `closed` folder, an Oracle copyright notice is expected, which is not appropriate for that file.

Reply:
It would make sense to move the .copyrightignore to the closed folder, but that means additional changes better left to other PRs.